src/js/profile/mobile/widget/Calendar.js

```diff
@@ -40,17 +44,43 @@
 				events = ns.event,
 				days = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"],
 				fullNameMonth = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
-				today = new Date(),
-				todayYear = today.getFullYear(),
-				todayMonth = today.getMonth() + 1,
-				defaultToday = today.getUTCDate(),
+				dateData = new Date(),
+				todayYear = dateData.getFullYear(),
+				todayMonth = dateData.getMonth() + 1,
+				defaultToday = dateData.getUTCDate(),
+				loadTodayFlag = true,
+				activeMonth,
+				fixMonth,
 				switchElement,
+				arrowElement,
 				selectDay,
+				classes = {
+					PREV_DISABLE_DAY: "prevDisableDay",
+					NON_SELECTION_DAY: "nonSelectionDay",
+					NEXT_DISABLE_DAY: "nextDisableDay",
+					CURRENT_ENABLE_DAY: "currentEnableDay",
+					SELECTION: "selection",
+					ARROW_RIGHT: "calendarArrowRight",
+					ARROW_LEFT: "calendarArrowLeft",
+					NON_SELECTION_ARROW: "ui-nonSelectionArrow",
+					NON_ARROW_LEFT: "calendarArrowLeft ui-nonSelectionArrow",
```

Comment:
jedna zmienna - jedna klasa

Reply:
"calendarArrowLeft ui-nonSelectionArrow" must always be present in the class.